install-dev-tool.sh

```
@@ -0,0 +1,40 @@
+
+if ! command -v dotnet >/dev/null 2>&1; then
+  echo "dotnet SDK (8+) is required to build and install cascode." >&2
+  exit 1
+fi
+
+mkdir -p "$nupkg_dir"
+rm -f "$nupkg_dir"/*.nupkg
+
+echo "Packing Cascode CLI..."
+dotnet pack "$csproj" -c Release -o "$nupkg_dir"
+nupkg=$(ls "$nupkg_dir"/Cascode.Cli.*.nupkg 2>/dev/null | head -n 1 || true)
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)